doc/UserGuide.md

```diff
@@ -34,14 +34,14 @@ Format: `help`
  
 ### Adding a person: `add`
 Adds a person to the address book<br>
-Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...` 
+Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/BLOCK, STREET, UNIT, POSTAL_CODE [t/TAG]...` 
```

Comment:
Good job here for updating the user guide as the format of input has changed. However, when you modify the behavior, remember to update the tests as well.